src/components/Message/components/Button/Button.js

```diff
@@ -20,7 +35,7 @@ const alignmentStyles = ({ align }) => {
     [TOP]: 'flex-start'
   };
   return css`
-    label: message__button--${align};
+    label: message__button-- ${align};
```

Comment:
eslint wrongfully fixes it, below is a workaround.

```suggestion
    label: ${`message__button--${align}`};
```